src/components/routeComponents/markAttendance/RoomAttendance.jsx

```diff
@@ -126,6 +134,76 @@ const RoomAttendance = ({ formConfig, roomAttendanceGridConfig, constants, apiCo
 		}
 	};
 
+	const updateOptionalAttendance = async ({ membersId }) => {
+		setShowSuccessAlert(false);
+		setShowErrorAlert(false);
+		if (!membersId.length) {
+			setShowErrorAlert(true);
+			setErrorMessage('Please add Ids first');
```

Comment:
This message should come from constants JSON
```suggestion
			setErrorMessage('Please add Ids first');
```